src/Runner.Sdk/ActionPlugin.cs

```diff
@@ -73,7 +73,7 @@ public VssConnection InitializeVssConnection()
         {
             var headerValues = new List<ProductInfoHeaderValue>();
             headerValues.Add(new ProductInfoHeaderValue($"GitHubActionsRunner-Plugin", BuildConstants.RunnerPackage.Version));
-            headerValues.Add(new ProductInfoHeaderValue($"({RuntimeInformation.OSDescription.Trim()})"));
+            headerValues.Add(new ProductInfoHeaderValue($"({StringUtil.SanitizeUserAgentHeader(RuntimeInformation.OSDescription)})"));
```

Comment:
Looks like we're already solving this a different way [elsewhere](https://github.com/actions/runner/blob/d0a4a41a632072512aee5aedc9c4430d50c06341/src/Sdk/WebApi/WebApi/Utilities/UserAgentUtility.cs#L39)?